SerialPrograms/Source/PokemonSV/Programs/Farming/PokemonSV_MaterialFarmerTools.cpp

```diff
@@ -121,18 +125,18 @@ MaterialFarmerOptions::MaterialFarmerOptions(
     , SAVE_DEBUG_VIDEO(
         "<b>DEV MODE: Save debug videos to Switch:</b><br>"
         "Set this on to save a Switch video everytime an error occurs. You can send the video to developers to help them debug later.",
-        LockMode::LOCK_WHILE_RUNNING,
+        LockMode::UNLOCK_WHILE_RUNNING,
         false
     )
     , SKIP_WARP_TO_POKECENTER(
         "<b>DEV MODE: Skip warping to closest PokeCenter:</b><br>"
         "This is for debugging the program without waiting for the initial warp.",
-        LockMode::LOCK_WHILE_RUNNING,
+        LockMode::UNLOCK_WHILE_RUNNING,
         false
     )
-    , SKIP_SANDWICH(
-        "<b>DEV MODE: Skip making sandwich:</b><br>"
-        "This is for debugging the program without waiting for sandwich making.",
+    , ENABLE_SANDWICH(
+        "<b>Enable Sandwich making:</b><br>"
+        "This is for boosting spawn rates of specific Pokemon.",
```

Comment:
You don't need a separate option to toggle the entire sandwich part. The `SandwichMakerOption` inherits from `GroupOption` which supports adding a checkbox to toggle the entire thing.

Lemme make a small change to `SandwichMakerOption` to pass this feature down.

The same thing applies to the entire material farmer in the Item Printer. I already merged it, but it uses a separate option to toggle it. You can also just defer to the `GroupOption` to show the checkbox instead.

Reply:
Lemme think about this. Since I haven't used the program yet. The reason why my line of thought went in this this direction is because I feel the sandwich/material farmer are add-ons to existing programs - not something that fundamentally changes their behavior.

Reply:
I'll merge this just to get it out of the way. But will probably tweak things later.